File: 2 ano/pc 2/Poligonos/src/poligonos/Poligonos.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poligonos;

/**
 *
 * @author JP
 */
public class Poligonos {
    private double base;
    private double altura;
    private double resultado;
    /**
     * @param base the base to set
     */
    public void setBase(double base) {
        this.base = base;
    }

    /**
     * @param altura the altura to set
     */
    public void setAltura(double altura) {
        this.altura = altura;
    }

    /**
     * @return the resultado
     */
    public double getResultado() {
        return resultado;
    }
    public void areaRetangulo(){
        resultado = base * altura;
    }
    public void areaTriangulo(){
        resultado = (base * altura)/2;
    }
    
    public void perimetroRetangulo(){
        resultado = (2*base) + (2*altura);
    } 
    
}

File: 2 ano/pc 1/Poligonos/Poligonos.vb
Public Class Poligonos
    Public Property base As Double
    Public Property altura As Double
    Public Property resultado As Double
    Public Sub areaRetangulo()
        resultado = base * altura
    End Sub
    Public Sub perimetroRetangulo()
        resultado = base + base + altura + altura
    End Sub
    Public Sub areaTriangulo()
        resultado = base * altura / 2
    End Sub
End Class
